docs/contributing/featureflags.md

```diff
@@ -124,7 +125,15 @@ the appropriate access. You should discuss:
 - The data type of the flag.
 - The default value of the flag.
 - The possible values of the flag (for non-boolean types).
-- Any context-based rules that should drive flag behavior.
+- Any context-based rules that should drive flag behavior. :::tip When should I request the flag in
+  LaunchDarkly?
+
+As a general rule, feature flag should be requested for creation in LaunchDarkly as part of merging
+the code using the flag into the `master` branch. Since local development and QA testing with their
+self-hosted instances will use local data sources, the first time that a flag in LaunchDarkly would
+be referenced is when the code is deployed to a cloud environment from the `master` branch.
```

Comment:
💭 Doesn't QA deploy feature branches sometimes? I realize this isn't the norm but unsure if we should mention that `master` branch specifically here.

Reply:
Good point.  I've updated to remove the reference to `master`.